contrib/chart/backstage/README.md

```diff
@@ -62,13 +62,31 @@ backstage-ingress   *       123.1.2.3       80      17m
 
 > **NOTE**: this is not a production ready deployment.
 
-## Caveats
-
-The current implementation does not generate certificates for the ingress which means the browser will alert that the
-site is insecure and using self-signed certificates.
 
 ## Customization
 
+### Issue certificates
+
+These charts can install or reuse a `clusterIssuer` to generate certificates for the backstage `ingress`. To do that:
+
+1. [Install][install-cert-manager] or make sure [cert-manager][cert-manager] is installed in the cluster.
+2. Enable the issuer in the charts. This will first check if there is a `letsencrypt` issuer already deployed in your
+   cluster and deploy one if it doesn't exist.
+
+To enable it you need to provide a valid email address in the chart's values:
+
+```
+issuer:
+  email: me@example.com
+  clusterIssuer: "letsencrypt-prod"
+```
+
+By default, the charts use `letsencrypt-staging` so in the above example we instruct helm to use the production issuer
+instead.
+
+[cert-manager]: https://cert-manager.io/docs/
+[cert-manager]: https://cert-manager.io/docs/installation/kubernetes/#installing-with-helm
```

Comment:
should this reference be `install-cert-manager`?

Reply:
Thanks! fixed now